sdk/tables/ci.yml

```diff
@@ -0,0 +1,66 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/tables
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/tables
+      - eng/common/testproxy
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+    parameters:
+      ServiceDirectory: tables
+      # Not running tests on CI but only on LiveTest mode
+      CtestRegex: azure-data
+      LineCoverageTarget: 63
+      BranchCoverageTarget: 33
+      LiveTestCtestRegex: azure-data
+      Clouds: Preview
+      SupportedClouds: Preview
+      Artifacts:
+        - Name: azure-data-tables
+          Path: azure-data-tables
+          VcpkgPortName: azure-data-tables-cpp
+      TestEnv:
+        - Name: STANDARD_STORAGE_CONNECTION_STRING
+          Value: "DefaultEndpointsProtocol=https;AccountName=account-name;AccountKey=3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;EndpointSuffix=core.windows.net"
+        - Name: PREMIUM_FILE_CONNECTION_STRING
```

Comment:
I don't see this being used anywhere in your samples, drop it?

Reply:
i'm not touching this section, things are to finicky and they blow up for no reason,